client/src/components/ContentNavigator/viya/DataDescriptor.ts

```diff
@@ -0,0 +1,88 @@
+// Copyright © 2022, SAS Institute Inc., Cary, NC, USA. All Rights Reserved.
```

Comment:
This was a sub class of an abstracted class which had another WIP descriptor. They're used to describes the item for different platforms. It had some internal attributes which are now useless or hard coded. If we're going to support this navigation pane only for Viya, it's actually unnecessary to declare this class or declared as a collection of utility functions.

Reply:
Good point. A class isn't really needed. I moved all functionality to utils.js and got rid of data descriptor